libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.GetConversationReference.schema

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "https://schemas.botframework.com/schemas/component/v1.0/component.schema",
+    "$role": "implements(Microsoft.IDialog)",
+    "title": "Get ConversationReference for current activity context.",
```

Comment:
>Get ConversationReference for current activity context.", [](start = 14, length = 57)

Pretty sure this is too long and shouldn't have a period at the end.  Probably just "Get ConversationReference"